### PR TITLE
chore: change the javascript in Github action to appropriate language

### PR DESCRIPTION
**Pull request by diegovanelli**
# Description

Change the javascript in Github action to appropriate language

Fixes https://github.com/Greenstand/treetracker-web-map-client/issues/1472

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by dadiorchen** [comment]
@diegovanelli great, did you test it?

**Comment by diegovanelli** [comment]
Hello!
Yes, I did.
But I saw a problem in the .releaserc.json file which has different pattern in the branch names. One option is to standardize as an object and set the name of each branch.

What do you think?

**Comment by dadiorchen** [comment]
@diegovanelli do you mean these two: {master}, and {name: 'cwm', channel}, this the default patten in semantic release, I hope we can support this standard format, by ignoring the extra properties like channel, it means the same name as the given one.

File: .github/workflows/dev-build-release.yaml [hunk 1]
```diff
@@ -27,7 +27,15 @@ jobs:
...
-        run: echo ::set-output name=RELEASE_NEEDED::$(echo `node -e 'console.log(require("./.releaserc.json").branches.map(e => e.name || e).reduce((a,c) => a || c === process.env.GITHUB_REF_NAME, false))'`)
+        run: |
+          RELEASE_NEEDED=false
+          while read -r branch; do
+            if [ "$branch" == "${GITHUB_REF_NAME}" ]; then
+              RELEASE_NEEDED=true
+              break
+            fi
+          done < <(jq -r '.branches[].name' '.releaserc.json')
+          echo "::set-output name=RELEASE_NEEDED::$RELEASE_NEEDED"
```
Comment (dadiorchen) [inline .github/workflows/dev-build-release.yaml:38]: I think bash isn't a better way comparing to JS, it is also problem prone, maybe we can find some Github Action to help on this, a component/plugin

**Pull request closed without merging**